Applications/SimpleShot/layers/centorid_nearest_neighbors.cpp

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   nearest_neighbors.cpp
+ * @date   09 Jan 2021
```

Comment:
I think we need to replace this layer with KNN.

Reply:
Yes, we will have to.

As this would require some bigger changes than creating custom layer, is it okay to leave as it is in this PR and later change if not urgent?